### PR TITLE
Fixes #67 : Make sure kafka producer is called when KafkaSender is cl…

### LINKED ISSUE #67: [Bug] Kafka producer is not closed when kafka11 sender is closed
Hi,

I realised that the kafka producer is no more a LazyCloseable. Therefore, the close method do not call the kafka producer close method and leaks into the Java MXBean registry.

I'll submit a test+fix soon.

Thanks

### PR DESCRIPTION
**Pull request by hypnoce**
…osed.

### CODE REVIEW

File: kafka11/src/main/java/zipkin2/reporter/kafka11/KafkaSender.java [hunk 1]
```diff
@@ -169,6 +170,10 @@ public final KafkaSender build() {
 
   @Override public void close() throws IOException {
     if (closeCalled) return;
+    final Closeable closeable = get();
```
Comment (adriancole) [inline kafka11/src/main/java/zipkin2/reporter/kafka11/KafkaSender.java]: I think it is impossible for this to return null

**Comment by adriancole** [comment]
doublechecked and I think this is the only sender with this bug. will address the small nit on the way in. Thanks!

**Comment by adriancole** [comment]
releasing immediately

**Comment by hypnoce** [comment]
Sorry I was sleeping :) Thanks ! (btw there is the Closeable import that you could remove :p)

**Pull request merged**